js/main.js

```diff
@@ -0,0 +1,261 @@
+// document.addEventListener('DOMContentLoaded', init)
+
+// var bs = burgerStack()
+
+// function init () {
+// userinput hamburger
+
+// setTimeout(function () {alert("DO READ INSTRUCTIONS BEFORE START")}, 1000)
+  var u1 = document.getElementById('u1')
```

Comment:
try to use `querySelector` instead of `getElementById`. We gonna use similar way of targeting using `jQuery` later